js/modules/k6/execution/execution.go

```diff
@@ -0,0 +1,169 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package execution
+
+import (
+	"context"
+	"errors"
+
+	"github.com/dop251/goja"
+
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/lib"
+)
+
+// Execution is a JS module to return information about the execution in progress.
+type Execution struct{}
+
+// New returns a pointer to a new Execution.
+func New() *Execution {
+	return &Execution{}
+}
+
+// GetVUStats returns information about the currently executing VU.
+func (e *Execution) GetVUStats(ctx context.Context) (goja.Value, error) {
+	vuState := lib.GetState(ctx)
+	if vuState == nil {
+		return nil, errors.New("getting VU information in the init context is not supported")
+	}
+
+	rt := common.GetRuntime(ctx)
+	if rt == nil {
+		return nil, errors.New("goja runtime is nil in context")
+	}
+
+	stats := map[string]interface{}{
+		"id":         vuState.VUID,
+		"idGlobal":   vuState.VUIDGlobal,
+		"idScenario": vuState.VUIDScenario,
+		"iteration":  vuState.Iteration,
+		"iterationScenario": func() goja.Value {
```

Comment:
```suggestion
		"idInScenario": vuState.VUIDScenario,
		"iteration":  vuState.Iteration,
		"iterationInScenario": func() goja.Value {
```

This sounds just a bit better to me, but it's a minor nitpick. @MStoykov @imiric WDYT?

Reply:
Sure, I'm fine with that. Just that `idGlobal` and `iterationGlobal` in `getScenarioStats()` would slightly stick out, and renaming them to something like `idInTest` would be a bit more confusing. We'll have to document each of these properly anyway, so having them shorter and following the same naming convention would be better UX. We'd also document them in the TypeScript definitions, so the name itself doesn't matter much.